### ISSUE TITLE
User is able to add spaces between words in confirm seed window

### ISSUE THREAD

**Issue by Anilkumar18** [Feathercoin-Foundation/electrum-ftc#26]
In confirm seed window , after pasting the seed in the confirm seed box , user is able to give spaces between word and click on next button

![seed_error](https://user-images.githubusercontent.com/32846534/40547120-a618b6d8-604e-11e8-8bb8-9c37980fa223.png)

Expected behaviour : Either an alert should be provided to user to not edit the original seed or the NEXT button provided in the page should be disabled when user adds/edits a seed

Note : Exists in Bitcoin also

**Comment by Anilkumar18** [comment]
Issue exists in Bitcoin also.

**Comment by lclc** [comment]
I don't think this is a problem. As long as it results in the same wallet (which it should).

Maybe you can try with importing a seed from a wallet that has some Bitcoins (/ a balance), adding some spaces to it. It should work and show the same balance in the new wallet.

**Comment by Anilkumar18** [comment]
Tried importing a seed from wallet that has balance by adding some spaces to it .
The wallet was restored showing the same balance
